### PR TITLE
Template update

### PR DESCRIPTION
**Pull request by vojtechrojicek**
Issue "Template update recommendations #5"

### CODE REVIEW

File: Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Application/Controllers/RRREntityNameRRR_Plural_Controller.cs [hunk 1]
```diff
@@ -12,13 +13,20 @@ namespace Kros.CqrsTemplate.Application.Controllers
     /// </summary>
     [Route("api/[controller]")]
     [ApiController]
+    [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(ValidationProblemDetails))]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     public class RRREntityNameRRR_Plural_Controller : ControllerBase
```
Comment (Burgyn) [inline Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Application/Controllers/RRREntityNameRRR_Plural_Controller.cs:20]: And what about `401 - Unauthorize`?

File: Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Application/Controllers/SecurityTestController.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace Kros.CqrsTemplate.Application.Controllers
+{
+    /// <summary>
+    /// SecurityTest controller.
+    /// </summary>
+    [Route("api/[controller]")]
+    [ApiController]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(ValidationProblemDetails))]
```
Comment (Burgyn) [inline Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Application/Controllers/SecurityTestController.cs]: By my opinion status codes `404` and `500` are misconceptions.
I would replace them with `401 - Unauthorize`

**Review by Burgyn — CHANGES_REQUESTED** [review]
Btw, for future: when you write comment something like "Closed #8" to PR then GitHub automaticaly close issue asociated with PR when PR is merged.

File: Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Application/Controllers/RRREntityNameRRR_Plural_Controller.cs (hunk 1 above)
Comment (vojtechrojicek) [reply Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Application/Controllers/RRREntityNameRRR_Plural_Controller.cs:20]: Fixed in next commit.

**Review by Burgyn — APPROVED** [review]

File: Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Application/Controllers/SecurityTestController.cs (hunk 2 above)
Comment (vojtechrojicek) [reply Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Application/Controllers/SecurityTestController.cs]: Aggreed. Fixed in next commit.

**Pull request merged**